src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressFilterElement.java

```diff
@@ -105,7 +105,7 @@ public SuppressFilterElement(String files, String checks,
             messageRegexp = null;
         }
         else {
-            messageRegexp = Pattern.compile(message);
+            messageRegexp = Pattern.compile(messagePattern);
```

Comment:
This will not make any cause in code as messagePattern is already equal to message